projects/vdk-plugins/vdk-lineage/src/vdk/plugin/lineage/sql_lineage_parser.py

```diff
@@ -74,7 +74,7 @@ def get_table_lineage_from_query(
 
     return LineageData(
         query=query,
-        query_type=runner.statements_parsed[0].get_type(),
+        query_type="not implemented",
```

Comment:
Is there no way to get the query type now?

Reply:
Not an obvious and easy way. I could use sqlparse library (which sqlilineage used previously) to get it but as it's not needed I opted not to do it now.

Reply:
Fair enough. You should open a ticket for it if you think it's something we'll really need in the future. I'm ok with resolving this either way. 🙂